src/Text/Pretty/Core.agda

```diff
@@ -142,8 +142,8 @@ private
 
     pad : Maybe String
     pad with x.lastWidth
-    ... | 0 = nothing
-    ... | l = just (replicate l ' ')
+    ... | 0         = nothing
```

Comment:
What about `if (does (NonZero? x.lastWidth)) then nothing else...`? Or even, as above, a use of `case_of_` here? 

I was puzzled by your keeping this use of `with`... given the nature of this PR!?

Reply:
This one is probably just a case of me being working on this file too late at night and just not seeing the obvious!

Reply:
Quite. (You are not alone...)